H2MLauncher.Core/ViewModels/ServerBrowserViewModel.cs

```diff
@@ -86,31 +88,40 @@ public ServerBrowserViewModel(
             CheckUpdateStatusCommand = new AsyncRelayCommand(CheckUpdateStatusAsync);
             CopyToClipBoardCommand = new RelayCommand<ServerViewModel>(DoCopyToClipBoardCommand);
             SaveServersCommand = new AsyncRelayCommand(SaveServersAsync);
-            UpdateLauncherCommand = new AsyncRelayCommand(DoUpdateLauncherCommand, () => _updateStatus != "" && !AutoUpdateFinished);
+            UpdateLauncherCommand = new AsyncRelayCommand(DoUpdateLauncherCommand, () => _updateStatus != "");
```

Comment:
Use `UpdateStatus` instead of field + ReleaseNotesVisibility because it has `NotifyCanExecuteChangedFor` thingy (better rename it back to `AutoUpdateFinished` or something like that, the name is confusing because it does not show release notes)